docs/proposals/16-adminkubeconfig-subresource.md

```diff
@@ -54,22 +54,22 @@ There are several problems with this approach:
 
 ## Proposal
 
-The `gardener-apiserver` would serve a new `shoots/adminkubeconfig` resource. It can only accept `CREATE` calls and accept `AdminKubeConfigRequest`. A `AdminKubeConfigRequest` would have the following structure:
+The `gardener-apiserver` would serve a new `shoots/adminkubeconfig` resource. It can only accept `CREATE` calls and accept `AdminKubeconfigRequest`. A `AdminKubeconfigRequest` would have the following structure:
 
 ```yaml
 apiVersion: authentication.gardener.cloud/v1alpha1
-kind: AdminKubeConfigRequest
+kind: AdminKubeconfigRequest
 spec:
   expirationSeconds: 3600
 ```
 
-Where `expirationSeconds` is the validity of the certificate in seconds. It this case it would be `1 hour`. The maximum validity of a `AdminKubeConfigRequest` is configured by `--shoot-admin-kubeconfig-max-expiration` flag in the `gardener-apiserver`.
+Where `expirationSeconds` is the validity of the certificate in seconds. It this case it would be `1 hour`. The maximum validity of a `AdminKubeconfigRequest` is configured by `--shoot-admin-kubeconfig-max-expiration` flag in the `gardener-apiserver`.
```

Comment:
```suggestion
Where `expirationSeconds` is the validity of the certificate in seconds. In this case it would be `1 hour`. The maximum validity of a `AdminKubeconfigRequest` is configured by `--shoot-admin-kubeconfig-max-expiration` flag in the `gardener-apiserver`.
```